tests/unit/algorithms/detection/tiling/test_tiling_detection_unittest.py

```diff
@@ -0,0 +1,172 @@
+        self.width = 1024
+        self.label_names = ["rectangle", "ellipse", "triangle"]
+        self.tile_cfg = dict(
+            tile_size=np.random.randint(low=100, high=500),
+            overlap_ratio=np.random.uniform(low=0.0, high=0.5),
+            max_per_img=np.random.randint(low=1, high=10000),
+        )
+        self.dataloader_cfg = dict(samples_per_gpu=1, workers_per_gpu=1)
+        self.otx_dataset, self.labels = create_otx_dataset(self.height, self.width, self.label_names)
+
+        img_norm_cfg = dict(mean=[0, 0, 0], std=[255, 255, 255], to_rgb=True)
+
+        self.train_data_cfg = ConfigDict(
+            dict(
+                type="ImageTilingDataset",
+                pipeline=[
+                    dict(type="Resize", img_scale=(self.height, self.width), keep_ratio=False),
+                    dict(type="RandomFlip", flip_ratio=0.5),
+                    dict(type="Pad", size_divisor=32),
+                    dict(type="DefaultFormatBundle"),
+                    dict(type="Collect", keys=["img", "gt_bboxes", "gt_labels"]),
+                ],
+                dataset=dict(
+                    type="MPADetDataset",
+                    pipeline=[
+                        dict(type="LoadImageFromOTXDataset"),
+                        dict(
+                            type="LoadAnnotationFromOTXDataset",
+                            with_bbox=True,
+                            with_mask=False,
+                            domain=Domain.DETECTION,
+                            min_size=-1,
+                        ),
+                    ],
+                    otx_dataset=self.otx_dataset,
+                    labels=self.labels,
+                    domain=Domain.DETECTION,
+                ),
+                **self.tile_cfg
+            )
+        )
+
+        self.test_data_cfg = ConfigDict(
+            dict(
+                type="ImageTilingDataset",
+                pipeline=[
+                    dict(
+                        type="MultiScaleFlipAug",
+                        img_scale=(self.height, self.width),
+                        flip=False,
+                        transforms=[
+                            dict(type="Resize", keep_ratio=False),
+                            dict(type="Normalize", **img_norm_cfg),
+                            dict(type="ImageToTensor", keys=["img"]),
+                            dict(type="Collect", keys=["img"]),
+                        ],
+                    )
+                ],
+                dataset=dict(
+                    type="MPADetDataset",
+                    pipeline=[dict(type="LoadImageFromOTXDataset")],
+                    otx_dataset=self.otx_dataset.with_empty_annotations(),
+                    labels=list(self.labels),
+                    domain=Domain.DETECTION,
+                ),
+                test_mode=True,
+                **self.tile_cfg
+            )
+        )
+
+    def test_tiling_train_dataloader(self):
```

Comment:
If it's a unit test, could you add a unit test decorator @e2e_pytest_unit?
`from tests.test_suite.e2e_test_system import e2e_pytest_unit`